ocis-pkg/log/log.go

```diff
@@ -17,6 +19,10 @@ import (
 
 var (
 	RequestIDString = "request-id"
+
+	// Match all paths outside of ocis. Will break if path does not include '/ocis/',
+	// but this is intended for debugging purposes.
+	pathRegex = regexp.MustCompile(`.*/ocis/`)
```

Comment:
Can we do it without this regex? Maybe we just log the filename and the linenumber?

Reply:
I agree, having filename/linenumbers in the log is useful no matter what.

Reply:
The call to `runtime.Caller` returns the full filename and path from where the file was when the binary was built, so that was what I was trying to avoid.

Reply:
But, is that really a problem? I don't think so. At least not at debug level.

Reply:
Can't we just use `filepath.Base` to extract only the file name? That shouldn't be too much overhead. https://pkg.go.dev/path/filepath#Base

Reply:
In that case I would prefer to have the full filename. We have so many `service.go` files ;-)

Reply:
Ok. Then let's go for full path for simplicity :+1:

Reply:
Done.